Formula/binaryen.rb

```diff
@@ -13,6 +13,7 @@ class Binaryen < Formula
   end
 
   depends_on "cmake" => :build
+  depends_on "python" => :build
```

Comment:
We're in the process of migrating formulae to python 3.8, but since this is only a build dependency, I think it's fine as is. Although it would be good to verify that this will build with `python@3.8` locally.

Reply:
Should be fine, they are using python to generate some code, so if it already works with Python 3.7 (which is the default `python`, right?), 3.8 should not be an issue.